iina/MainWindowController.swift

```diff
@@ -1576,7 +1576,26 @@ class MainWindowController: PlayerWindowController {
       window?.title = player.getMediaTitle()
     } else {
       window?.representedURL = player.info.currentURL
-      window?.setTitleWithRepresentedFilename(player.info.currentURL?.path ?? "")
+      // Workaround for issue #3543, IINA crashes reporting:
```

Comment:
Please link a feedback number, if any

Reply:
I added the feedback number. It was filed back on Nov 30th. I don't think Apple has looked at it.